examples/kfserving/main.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import fairing
+
+if __name__ == '__main__':
+        fairing.config.set_builder(name='docker', registry='<Your-Docker-Registry>',
```

Comment:
do you need a builder here?

Reply:
I set the builder since I met the connecting exceptions with gcr.io because of GOOGLE_APPLICATION_CREDENTIALS no configuration. If set the my registry, will no the problem, so I just suggest keep this since other users may met the problem. Agree? @karthikv2k Thanks.

Reply:
We should write this example by directly calling the Serving class. There is no need to preprocess or build steps here.

Reply:
Removed builder here. Thanks.